manifests/a/Almico/SpeedFan/4.52/Almico.SpeedFan.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Almico.SpeedFan
+PackageVersion: 4.52
+Installers:
+- Architecture: x86
+  InstallerType: nullsoft
+  InstallerUrl: https://www.almico.com/speedfan452.exe
+  InstallerSha256: E2CCB3C0D23F0D04EE8057F5CE3861EEA952FB20694C1656C9805B1D4CD922FF

```

```suggestion
  InstallerSha256: E2CCB3C0D23F0D04EE8057F5CE3861EEA952FB20694C1656C9805B1D4CD922FF
  ProductCode: SpeedFan
```
for mapping in `winget list`